### PR TITLE
Broken changelog page

### PR DESCRIPTION
**Pull request by robertguss**
- fixes the broken changelog page.

### CODE REVIEW

**Comment by robertguss** [comment]
@jaffrepaul so I manage to fix the changelog page partially. The issue I cannot figure out is how to sort the changelog orders in desc order so the latest release is first. It looks like we can do it, via these docs, but I cannot figure it out so maybe you or @elylucas can give it a shot. 

https://docusaurus.io/docs/sidebar/autogenerated#customize-the-sidebar-items-generator

**Comment by admah** [comment]
@jaffrepaul nice work getting this together! I haven't had a chance to look at the PR yet, but I see 2 things that are different than the current implementation: 

1. We aren't setting an "active" class on the heading being viewed.
2. We aren't scrolling down far enough. The version number heading is being cut off currently.

This is the current changelog: 

Screen Shot 2022-09-20 at 12 47 07 PM

File: docs/guides/references/_changelog-list.mdx [hunk 1]
```diff
@@ -0,0 +1,284 @@
+## 10.7.0
```
Comment (admah) [inline docs/guides/references/_changelog-list.mdx:1]: Is this file auto-generated or will updating this need to be a part of the release process?

File: docs/guides/references/changelog.mdx [hunk 2]
```diff
@@ -2,4 +2,10 @@
 title: Changelog
 ---
 
-{% changelog %}
+<!-- Docusaurus has a few open issues with mdx and the TOC where the TOC doesn't parse/display dynamically like below without scanning the markdown directly. So we use the workaround noted here for now https://github.com/facebook/docusaurus/pull/5331 -->
```
Comment (admah) [inline docs/guides/references/changelog.mdx:5]: Good reference!

File: docusaurus.config.js [hunk 3]
```diff
@@ -67,8 +68,36 @@ const config = {
 
   plugins: [
     './plugins/fav-icon',
+    async function myPlugin(context, options) {
```
Comment (admah) [inline docusaurus.config.js:71]: Is it ok to rename this function to something more specific?

**Review by admah — CHANGES_REQUESTED** [review]
Great work on this. I had a couple of questions in addition to my style comments from yesterday.

File: docusaurus.config.js (hunk 3 above)
Comment (jaffrepaul) [reply docusaurus.config.js]: yep!

File: docs/guides/references/_changelog-list.mdx (hunk 1 above)
Comment (jaffrepaul) [reply docs/guides/references/_changelog-list.mdx:2]: It's not autogenerated atm. That's a cool thing to add but it's sadly a part of the hack to get this TOC on the page.

**Comment by jaffrepaul** [comment]
> @jaffrepaul nice work getting this together! I haven't had a chance to look at the PR yet, but I see 2 things that are different than the current implementation:
> 
> 1. We aren't setting an "active" class on the heading being viewed.
> 2. We aren't scrolling down far enough. The version number heading is being cut off currently.
> 
> This is the current changelog:
> 
> [Screen Shot 2022-09-20 at 12 47 07 PM](https://user-images.githubusercontent.com/2184114/191340794-19b7cdf4-5d21-4a18-9a20-a99e8445109e.png)
> 
> This site:
> 
> [Screen Shot 2022-09-20 at 1 55 39 PM](https://user-images.githubusercontent.com/2184114/191341442-eac04c53-8aae-47b9-9ef6-c99815361a4f.png)

Cool. I solved the scroll issue. The active class has oddly proven to be difficult since its part of the TOC hack.

**Pull request merged**